src/QMCDrivers/CloneManager.cpp

```diff
@@ -21,7 +21,7 @@
 #include "Message/Communicate.h"
 #include "Message/OpenMP.h"
 #include "Utilities/IteratorUtility.h"
-#include <qmc_common.h>
+#include <Utilities/qmc_common.h>
```

Comment:
Quotes again? How is this different from the usage in qmcapp.cpp

Reply:
Will be fixed in a separate PR.